### ISSUE TITLE
[CMIP6 CMOR-ization & ESGF-publication] NorESM2-LM - ssp370SST-lowAer

### ISSUE THREAD

**Issue by DirkOlivie** [NorESMhub/noresm2cmor#185]
### Mandatory information:

**Full path to the case(s) of the experiment on NIRD**
/projects/NS9560K/noresm/cases

**experiment_id**
ssp370SST-lowAer

**model_id**
NorESM2-LM

**CASENAME(s) and years to be CMORized**
NFSSP370frc2norbc_aerlow_f19_20200110, 2015-2100

### Optional information

**parent_experiment_id**
histSST

**parent_experiment_rip**
r1i1p1f1

**parent_time_units**
'days since 1850-01-01'

**branch_method**
'Hybrid-restart from year 2015-01-01 of histSST'

**other information**

**Comment by YanchunHe** [comment]
@DirkOlivie Please open read access to these experiments, thanks!

**Comment by YanchunHe** [comment]
And according to the CMIP6 controlled vocabulary [CMIP6_CV.json](https://github.com/PCMDI/cmip6-cmor-tables/blob/master/Tables/CMIP6_CV.json), the `parent_experiment_id` is `historical` instead of `histSST`.

**Comment by DirkOlivie** [comment]
I have opened read-access now to NFSSP370frc2norbc_f19_20200110 and NFSSP370frc2norbc_aerlow_f19_20200110.

Concerning the parent : I have used restart dumps (2015-01-01) from histSST to start ssp370SST and ssp370SST-lowAer.  That was maybe not what was recommended, but I think it does not have a big impact on the results.  @YanchunHe, what do you think is best to write then as parent?

**Comment by YanchunHe** [comment]
Thanks!

I set `parent_experiment_id=historical`, since this has to comply with the CMIP6_CV.json, otherwise, cmor lib will throw a warming and reset it to `historical`. But I set `branch_method = 'Hybrid-restart from year 2015-01-01 of histSST'`, as this text value can be arbitrary.

The resulting global attribute will be:
```
// global attributes:
                :activity_id = "AerChemMIP" ;
                :branch_method = "Hybrid-restart from year 2015-01-01 of histSST" ;
                :experiment = "SSP3-7.0, with SSTs prescribed from ssp370" ;
                :experiment_id = "ssp370SST" ;
                 ...
                :parent_activity_id = "CMIP" ;
                :parent_experiment_id = "historical" ;
```

**Comment by YanchunHe** [comment]
cmorized and can be published to ESGF

**data path**
- /projects/NS9034K/CMIP6/.cmorout/NorESM2-LM/ssp370SST-lowAer
- /projects/NS9034K/CMIP6/AerChemMIP/NCC/NorESM2-LM/ssp370SST-lowAer/

**version**
- v20200218

**sha256sum**
- /projects/NS9034K/CMIP6/AerChemMIP/NCC/NorESM2-LM/ssp370SST-lowAer/.r1i1p1f1.sha256sum_v20200218

**Comment by monsieuralok** [comment]
@YanchunHe published

**Comment by YanchunHe** [comment]
Cmorized with additional iLAMB variables (#262), AERday `zg500` (#263) and corrected `fNup` (#251).

They are ready to be published to ESGF.

**data path**
- /projects/NS9034K/CMIP6/.cmorout/NorESM2-LM/ssp370SST-lowAer
- /projects/NS9034K/CMIP6/AerChemMIP/NCC/NorESM2-LM/ssp370SST-lowAer

**version**
- v20210203

**sha256sum**
/projects/NS9034K/CMIP6/AerChemMIP/NCC/NorESM2-LM/ssp370SST-lowAer
- .r1i1p1f1.sha256sum_v20210203

**Comment by monsieuralok** [comment]
@YanchunHe published

**Comment by YanchunHe** [comment]
New dataset version to fix issues #270, #271, #272 and #273 is ready to be published:

**data path**
- /projects/NS9034K/CMIP6/AerChemMIP/NCC/NorESM2-LM/ssp370SST-lowAer
- /projects/NS9034K/CMIP6/.cmorout/NorESM2-LM/ssp370SST-lowAer

**version**
- v20210319

**sha256sum**
- /projects/NS9034K/CMIP6/AerChemMIP/NCC/NorESM2-LM/ssp370SST-lowAer
- .r1i1p1f1.sha256sum_v20210319

**Comment by monsieuralok** [comment]
@YanchunHe published
